tox.ini

```diff
@@ -27,4 +27,4 @@ commands = pydocstyle
 [pydocstyle]
 match-dir = importer
 match = (?![test_|__init__|user\-config]).*\.py
-ignore = D100,D101,D102,D103
+ignore = D100,D101,D102,D103,D203,D212,D404
```

Comment:
Don't trim the last newline (if your editor does this by itself then it might be worth turning that off).